### DISCUSSION TITLE
News API

### DISCUSSION THREAD

**Discussion by Lee1887** [neuefische/questions#47] [Web] [answered]
Hello, I have a general question.
Does anyone have any connections to German publications (free API`s)  besides Bild?
So far, I’ve only seen American ones (CNN, ESPN;...).

I require them to populate my news page.

Thank you really much:)

**Comment by pixelass** [comment, marked as answer]
You could try to work with the twitter API and then use the tweets from i.e. ZDF. 

https://twitter.com/zdfheute
https://developer.twitter.com/en/docs/twitter-api
